### PR TITLE
ExceptionsCache accounts for string too-many-points error

### PR DESCRIPTION
**Pull request by bvaughn**
See [Replay 59207f3f-ae40-4fa3-8dd9-7aaffc601ece](https://app.replay.io/recording/console-doesnt-load-for-replay-with-large-number-of-exceptions--59207f3f-ae40-4fa3-8dd9-7aaffc601ece) for an explanation of _why_ this was failing. See [this Loom](https://www.loom.com/share/be7cd5be333a42928296f779490f97ee) and [Replay 03fa7e31-090c-4edc-b3f3-617fb1b9cd76](https://app.replay.io/recording/console-exceptions-fix--03fa7e31-090c-4edc-b3f3-617fb1b9cd76) for a demo of it working now.

Basically this was a case of [BAC-2330](https://linear.app/replay/issue/BAC-2330) burning us again. I also noticed and filed [BAC-2484](https://linear.app/replay/issue/BAC-2484) about some weird hit counts and Pause steps issue with the bug report Replay.

### CODE REVIEW

**Comment by bvaughn** [comment]
This is a pretty bad bug so I'm going to just land this fix and we can review it post.
